docs/flow/modin/experimental/xgboost.rst

```diff
@@ -58,54 +59,49 @@ internal implementation is the following:
-   :py:func:`~modin.experimental.xgboost.xgboost_ray._predict`) is split between actors evenly. The internal function
+4. Data `dtrain` is split between actors evenly. The internal function
    :py:func:`~modin.experimental.xgboost.xgboost_ray._split_data_across_actors` runs assigning row partitions to actors
    using internal function :py:func:`~modin.experimental.xgboost.xgboost_ray._assign_row_partitions_to_actors`.
-   This function creates a dictionary in the form: `{actor_rank: ([part_i0, part_i3, ..], [0, 3, ..]), ..}` for training,
-   `{actor_rank: [part_i0, part_i1, ..], ..}` for prediction.
+   This function creates a dictionary in the form: `{actor_rank: ([part_i0, part_i3, ..], [0, 3, ..]), ..}`.
 
 .. note:: :py:func:`~modin.experimental.xgboost.xgboost_ray._assign_row_partitions_to_actors` takes into account IP
   addresses of row partitions of `dtrain` data to minimize excess data transfer.
 
-1. For each :py:class:`~modin.experimental.xgboost.xgboost_ray.ModinXGBoostActor` the object methods ``set_train_data`` or ``set_predict_data`` are
-   called remotely. Those methods run by loading row partitions in actor according to the dictionary with partitions
+5. For each :py:class:`~modin.experimental.xgboost.xgboost_ray.ModinXGBoostActor` the object method ``set_train_data`` is
+   called remotely. This method runs loading row partitions in actor according to the dictionary with partitions
    distribution from previous step. When data is passed to the actor, the row partitions are automatically materialized
    (``ray.ObjectRef`` -> ``pandas.DataFrame``).
 
-2. Methods ``train`` or ``predict`` of :py:class:`~modin.experimental.xgboost.xgboost_ray.ModinXGBoostActor` class object are called remotely.
+6. Method ``train`` of :py:class:`~modin.experimental.xgboost.xgboost_ray.ModinXGBoostActor` class object is called remotely. This method
+   runs XGBoost training on local data of actor, connects to ``Rabit Tracker`` for sharing training state between
+   actors and returns dictionary with `booster` and `evaluation results`.
+
+7. On the final stage results from actors are returned. `booster` and `evals_result` is returned using ``ray.get``
+   function from remote actor.
+
+
+Prediction
+**********
 
-   * ``train``: method runs XGBoost training on local data of actor, connects to ``Rabit Tracker`` for sharing
-     training state between actors and returns dictionary with `booster` and `evaluation results`.
-   * ``predict``: method runs XGBoost prediction on local data of actor and returns IP address of actor and partial
-     prediction (``pandas.DataFrame``).
+1. The data is passed to :py:func:`~modin.experimental.xgboost.xgboost_ray._predict`
+   function as a :py:class:`~modin.experimental.xgboost.DMatrix` object.
 
-3. On the final stage results from actors are returned.
+2. The function :py:func:`~modin.experimental.xgboost.xgboost_ray._map_predict` is applied remotely for an each partition
```

Comment:
```suggestion
2. The function :py:func:`~modin.experimental.xgboost.xgboost_ray._map_predict` is applied remotely for each partition
```

Reply:
The point 2 was changed, check again please